monai/handlers/tensorboard_handlers.py

```diff
@@ -91,8 +91,8 @@ def __init__(
         self,
         summary_writer: SummaryWriter | SummaryWriterX | None = None,
         log_dir: str = "./runs",
-        iteration_log: bool = True,
-        epoch_log: bool = True,
+        iteration_log: bool | Callable[[Engine, int], bool] = True,
+        epoch_log: bool | Callable[[Engine, int], bool] = True,
         epoch_event_writer: Callable[[Engine, Any], Any] | None = None,
         epoch_interval: int = 1,
```

Comment:
I would suggest to add "deprecated" mark for `epoch_interval` and `iteration_interval` args, refer to:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/data/image_reader.py#L883
@wyli @ericspod What do you think?

Thanks.

Reply:
Hi @vfdev-5 ,

Could you please help update it? Others look good to me.

Thanks.

Reply:
Done